cli/index.js

```diff
@@ -81,4 +431,425 @@ y.command({
     }
 })
 
+// create login command
+y.scriptName("connectme")
+y.command({
+    command: 'login',
+    describe: 'login with an account',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        password: {
+            describe: 'Password of the account'

```

Comment:
I feel this also needs those two options no? demandOption: true, and type: 'string'

Reply:
done